python-threatexchange/setup.py

```diff
@@ -57,7 +57,7 @@
     ],
     keywords="facebook threatexchange",
     url="https://www.github.com/facebook/ThreatExchange",
-    packages=find_packages(exclude=["tests*"]),
+    packages=find_packages(exclude=["tests*"]),  # TODO also exclude subdir tests
```

Comment:
Do we have evidence that this does not exclude test packages? 

Why would subdir tests add new packages. New modules I understand, but packages?

Reply:
Maybe I misunderstood an error message I've seen, but I thought threatexchange/foo/tests stuff was being vacuumed into the package